modules/azure/function_app_linux_managed_identity/main.tf

```diff
@@ -61,50 +65,30 @@ resource "azurerm_linux_function_app" "function_app" {
     }
   }
 
+  auth_settings_v2 {
+    auth_enabled           = true
+    require_authentication = var.authentication_settings.require_authentication == null ? false : var.authentication_settings.require_authentication
+    unauthenticated_action = var.authentication_settings.unauthenticated_action == null ? null : var.authentication_settings.unauthenticated_action
+    excluded_paths         = var.authentication_settings.excluded_paths == null ? [] : var.authentication_settings.excluded_paths
+
+    active_directory_v2 {
+      client_id                  = local.should_create_app ? azuread_application.application[0].application_id : var.managed_identity_provider.existing.client_id
+      client_secret_setting_name = "MICROSOFT_PROVIDER_AUTHENTICATION_SECRET"
+      tenant_auth_endpoint       = "https://login.microsoftonline.com/v2.0/${data.azurerm_client_config.current.tenant_id}/"
+      allowed_audiences          = local.allowed_audiences
+    }
+
+    login {
+      // Bug within terraform module it just requires it
```

Comment:
By any chance is there an ticket of this bug in for example https://github.com/hashicorp/terraform-provider-azurerm?
If so link the ticket in the comment so we can easily see if the bug has been fixed.

Reply:
Added commented link to a issue I created